specification/metadata.md

```diff
@@ -10,5 +10,5 @@ abstract: |
           This will enable repository interoperability and assist in the
           management of the preservation of digital content.
 version: 2.0.4
-date: 01.06.2020
+date: 11.06.2020
```

Comment:
It says 12 on all the others

Reply:
Yep, cos that's right and this is wrong, thanks @karinbredenberg  :)